iotile_ext_cloud/iotile/cloud/cloud.py

```diff
@@ -88,6 +89,14 @@ def device_info(self, device_id):
 
         return dev
 
+    @return_type("bool")
+    def check_time(self):
```

Comment:
I'd take a parameter here `max_slop` or something similar with a default value of 300 that allows callers to override what is considered too much difference and removes the need to have a magic 300 in the comparison line